chunky/src/java/se/llbit/chunky/ui/RenderCanvasFx.java

```diff
@@ -247,6 +247,7 @@ public RenderCanvasFx(ChunkyFxController chunkyFxController,
         double halfWidth = canvas.getWidth() / (2.0 * canvas.getHeight());
         target.set(-halfWidth + event.getX() * invHeight,
             -0.5 + event.getY() * invHeight);
+        contextMenu.setUserData(new Vector2(event.getX(), event.getY()));
```

Comment:
Any reason not to use properties? imo it's more readable
```suggestion
        contextMenu.getProperties().put("canvasPos", new Vector2(event.getX(), event.getY()));
```

Reply:
Didn't know you could do that 👍